sdk/releases.md

```diff
@@ -0,0 +1,36 @@
+# Ably SDK Team: Guidance on Releases
+
+This document outlines our ideal requirements for SDK releases.
+
+There will be some SDK repositories:
+
+- which are yet to have the work done on them which will implement this
+- where it's necessary to deviate from some of the ideals specified here due to constraints which are innate or otherwise idiomatic to the platform being released for
+
+Where technically possible, SDK repository release processes which claim to adhere to this guidance must follow these ideal requirements closely.
+
+## Release Process
+
+1. Merge all pull requests containing changes intended for this release to `main` branch
+2. Prepare a [Release Branch](#release-branch) and a corresponding pull request, obtain approval from reviewers and then merge to `main` branch
+3. Trigger the [Publish Workflow](#publish-workflow)
+4. If the publish fails then add commits to the `main` branch that aim to fix the problem and then try step 3 again
+5. Push Git version tag, which may or may not have a `v` prefix according to the existing conventions for this SDK repository
```

Comment:
Is it worth just standardising this one way or the other? Or or there some specific reasons why some SDKs require the prefix, and others don't?

Reply:
Honestly, I don't know. This is all stuff that was put in place before I started working for Ably, so there's history needs digging into. I would _love_ to conform everywhere if possible - it's just sometimes impossible to tell what downstream dependencies might be reliant on current naming conventions per repository.

Reply:
Ok, so is it worth at least having an opinion here which one is more preferred so we're at least converging on one option going forwards?

Reply:
https://github.com/ably/engineering/pull/23/commits/edc9d8d54d48ffeb42fcf959ff253cae12fd41af

Consensus was clear from the poll that we should prefer the `v` prefix. As @owenpearson pointed out [in that internal Slack thread](https://ably-real-time.slack.com/archives/C030C5YLY/p1647963277793149?thread_ts=1647963003.830159&cid=C030C5YLY):

> it makes sense to take conventions from the creator of git himself https://github.com/torvalds/linux/tags